app/controllers/application_controller.rb

```diff
@@ -11,19 +11,24 @@ def switch_locale(&action)
   end
 
   def after_sign_in_path_for(resource)
-    user_path(resource)
+    @user = User.find(resource.id)
```

Comment:
Profile should be built after create user. Not after login
See https://gorails.com/forum/create-a-user-profile-after-saving-a-devise-user